### PR TITLE
Update Amazon SQS connector doc

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

### CODE REVIEW

File: en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+It has a single HTTP API resource, which is `sendToQueue`. 
+
+<img src="{{base_path}}/assets/img/integrate/connectors/amazonsqs-connector.jpg" title="AmazonSQS-Connector" width="800" alt="AmazonSQS-Connector"/>
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Set up the environment 
+
+1. Please follow the steps mentioned in the [Setting up the Amazon S3 Environment ]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document in order to create an Amazon account and obtain access key id and secret access key. Keep them saved to be used in the next steps.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md:17]:
```suggestion
1. Please follow the steps mentioned in the [Setting up the Amazon S3 Environment]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document to create an Amazon account and obtain access key ID and secret access key. Keep them saved to be used in the next steps.  
```

File: en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+<img src="{{base_path}}/assets/img/integrate/connectors/amazonsqs-connector.jpg" title="AmazonSQS-Connector" width="800" alt="AmazonSQS-Connector"/>
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Set up the environment 
+
+1. Please follow the steps mentioned in the [Setting up the Amazon S3 Environment]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document to create an Amazon account and obtain access key ID and secret access key. Keep them saved to be used in the next steps.  
+
+2. In this example we will be using XPath 2.0 which needs to be enabled in the product as shown below before starting the integration service.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md:19]:
```suggestion
2. In this example, we will be using XPath 2.0 which needs to be enabled in the product as shown below before starting the integration service. 
```

File: en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+
+It has a single HTTP API resource, which is `sendToQueue`. 
+
+<img src="{{base_path}}/assets/img/integrate/connectors/amazonsqs-connector.jpg" title="AmazonSQS-Connector" width="800" alt="AmazonSQS-Connector"/>
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Set up the environment 
+
+1. Please follow the steps mentioned in the [Setting up the Amazon S3 Environment]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document to create an Amazon account and obtain access key ID and secret access key. Keep them saved to be used in the next steps.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md:17]:
```suggestion
1. Follow the steps mentioned in the [Setting up the Amazon S3 Environment]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document to create an Amazon account and obtain access key ID and secret access key. Keep them saved to be used in the next steps.  
```

File: en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Set up the environment 
+
+1. Follow the steps mentioned in the [Setting up the Amazon S3 Environment]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document to create an Amazon account and obtain access key ID and secret access key. Keep them saved to use in the next steps.  
+
+2. In this example, we will be using XPath 2.0 which needs to be enabled in the product as shown below before starting the integration service. 
+
+    Add the following to the `PRODUCT-HOME/conf/deployment.toml` file. You can further refer to the [Product Configurations]({{base_path}}/reference/config-catalog-mi/#message-mediation).
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md:21]:
```suggestion
    Add the following to the `<PRODUCT_HOME>/conf/deployment.toml` file. You can further refer to the [Product Configurations]({{base_path}}/reference/config-catalog-mi/#message-mediation).
```

File: en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+
+    Add the following to the `<PRODUCT_HOME>/conf/deployment.toml` file. You can further refer to the [Product Configurations]({{base_path}}/reference/config-catalog-mi/#message-mediation).
+    
+      ```
+        [mediation]
+        synapse.enable_xpath_dom_failover="true"
+      ```
+
+    ??? note "Click here for instructions on configuring WSO2 Enterprise Integrator 6"
+        1. Uncomment the `synapse.xpath.dom.failover.enabled=true` property in PRODUCT-HOME/conf/synapse.properties file.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md:29]:
```suggestion
        1. Uncomment the `synapse.xpath.dom.failover.enabled=true` property in the `<PRODUCT_HOME>/conf/synapse.properties` file.
```

File: en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+               <progressionFactor>1</progressionFactor>
+           </suspendOnFailure>
+           <markForSuspension>
+               <retriesBeforeSuspension>0</retriesBeforeSuspension>
+           </markForSuspension>
+       </address>
+   </endpoint>
+   ```
+6. Then create the following sequences: `buildMessage`, `createQueue`, `sendMessage`, and `ReceiveAndForwardMessage`.
+7. Navigate to **MI Project Explorer** > **Sequences** and click on the **+** sign next to Sequences to open the **Create New Sequence** form.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md:94]:
```suggestion
7. Navigate to **MI Project Explorer** > **Sequences** and click the **+** sign next to **Sequences** to open the **Create New Sequence** form.
```

File: en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+    <sequence name="sendMessage" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+        <amazonsqs.sendMessage configKey="AMAZON_SQS_CONNECTION">
+            <queueId>{$ctx:queueId}</queueId>
+            <queueName>{$ctx:queueName}</queueName>
+            <messageBody>{$ctx:target_property}</messageBody>
+        </amazonsqs.sendMessage>
+    </sequence>
+  ```
+
+5. Create the ReceiveAndForwardMessage sequence as shown below. In this sequence, we will receive the message from the Amazon SQS queue and forward it to the StockQuote Endpoint.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md:152]:
```suggestion
5. Create the `ReceiveAndForwardMessage` sequence as shown below. In this sequence, we will receive the message from the Amazon SQS queue and forward it to the `StockQuote` Endpoint. 
```

File: en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md [hunk 8]
```diff
@@ -0,0 +1,272 @@
...
+<a href="{{base_path}}/assets/attachments/connectors/amazonSQSExample1.zip">
+    <img src="{{base_path}}/assets/img/integrate/connectors/download-zip.png" width="200" alt="Download ZIP">
+</a>
+
+!!! tip
+    You may need to update the value of the access key and make other such changes before deploying and running this project.
+
+## Deployment
+
+Follow the steps in [deploy-artifacts]({{base_path}}/develop/deploy-artifacts) guide to build and export it into a CAR file (.car file) using the WSO2 Micro Integrator Visual Studio Code extension.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md:214]:
```suggestion
Follow the steps in [deploy-artifacts]({{base_path}}/develop/deploy-artifacts) guide to build and export it into a CAR file (`.car` file) using the Micro Integrator for Visual Studio Code extension.
```

File: en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md [hunk 9]
```diff
@@ -0,0 +1,272 @@
...
+Follow the steps in [deploy-artifacts]({{base_path}}/develop/deploy-artifacts) guide to build and export it into a CAR file (`.car` file) using the Micro Integrator for Visual Studio Code extension.
+
+## Test
+
+1. Deploy the back-end service `SimpleStockQuoteService`.
+
+    1. Download the ZIP file of the back-end service from [here](https://github.com/wso2-docs/WSO2_EI/blob/master/Back-End-Service/axis2Server.zip).
+    2. Extract the downloaded zip file.
+    3. Open a terminal and navigate to the `axis2Server/bin/` directory inside the extracted folder.
+    4. Execute the following command to start the axis2server with the SimpleStockQuote back-end service:
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md:223]:
```suggestion
    4. Execute the following command to start the axis2server with the `SimpleStockQuote` back-end service:
```

File: en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md [hunk 10]
```diff
@@ -0,0 +1,272 @@
...
+       --header 'Content-Type: application/json' \
+       --data '{
+          "companyName":"WSO2",
+          "queueName":"Queue1"
+       }'
+    ```
+
+**Expected Response**:
+
+You should get the following response with the 'sys_id' and keep it saved.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md:247]:
```suggestion
Next, you will receive a response with `sys_id`. Make sure to save it."
```

File: en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md [hunk 11]
```diff
@@ -0,0 +1,272 @@
...
+        <ax21:prevClose>169.76483039627422</ax21:prevClose>
+        <ax21:symbol>WSO2</ax21:symbol>
+        <ax21:volume>19325</ax21:volume>
+    </ns:return>
+</ns:getQuoteResponse>
+```
+
+## What's next
+
+* To customize this example for your own scenario, see [Amazon SQS Connector Reference Guide]({{base_path}}/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-reference/) documentation for all operation details of the connector.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md:272]:
```suggestion
* To customize this example for your own scenario, see the [Amazon SQS Connector Reference Guide]({{base_path}}/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-reference/) documentation for detailed information on all connector operations.
```

File: en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-reference.md [hunk 12]
```diff
@@ -598,15 +552,376 @@ To use the Amazon SQS connector, add the <amazonsqs.init> element in your config
...
+    ```xml
+    <amazonsqs.createQueue configKey="AMAZON_SQS_CONNECTION">
+        <queueName>Queue</queueName>
+        <attributeEntries>Attribute.1.Name=DelaySeconds&amp;Attribute.1.Value=45</attributeEntries>
+    </amazonsqs.createQueue>
+    ```
+---
+
+??? note "deleteQueue"
+    This operation deletes the queue specified by the QueueUrl. See the [related API documentation](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_DeleteQueue.html) for more information.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-reference.md:653]:
```suggestion
    This operation deletes the queue specified by `QueueUrl`. See the [related API documentation](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_DeleteQueue.html) for more information.
```

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md [hunk 13]
```diff
@@ -12,29 +12,42 @@ It has a single HTTP API resource, which is `sendToQueue`.
...
-3. In this example we use the SimpleStockQuote service backend. Therefore, the SimpleStockQuote service needs to be started. 
-
-## Set up the Integration Project
-
-{!includes/build-and-run.md!}
-
-## Creating the Integration Logic
+## Set up the environment 
+
+1. Please follow the steps mentioned in the [Setting up the Amazon S3 Environment ]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document in order to create an Amazon account and obtain access key id and secret access key. Keep them saved to be used in the next steps.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md:17]:
```suggestion
1. Follow the steps mentioned in the [Setting up the Amazon S3 Environment]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document to create an Amazon account and obtain access key ID and secret access key. Keep them saved to use in the next steps.
```

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md [hunk 14]
```diff
@@ -0,0 +1,1058 @@
...
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>apiCallAttemptTimeout</td>
+            <td>The amount of time(in seconds) to wait for the http request to complete before giving up and timing out.</td>
+            <td>No</td>
+        </tr>
+    </table>
+
+    > **Note**: You can either pass credentials within init configuration or set the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY as environment variables. The AWS SDK uses provider chains to look for AWS credentials in system/user environment variables.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md:76]:
```suggestion
    > **Note**: You can either pass credentials within the `init` configuration or set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` as environment variables. The AWS SDK uses provider chains to look for AWS credentials in system/user environment variables.
```

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md [hunk 15]
```diff
@@ -0,0 +1,1058 @@
...
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>apiCallAttemptTimeout</td>
+            <td>The amount of time(in seconds) to wait for the http request to complete before giving up and timing out.</td>
+            <td>No</td>
+        </tr>
+    </table>
+
+    > **Note**: You can either pass credentials within the `init` configuration or set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` as environment variables. The AWS SDK uses provider chains to look for AWS credentials in system/user environment variables.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md:76]: Please include this note as an [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=) to ensure consistency within the documentation.

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md [hunk 16]
```diff
@@ -0,0 +1,1058 @@
...
+    
+        export AWS_ACCESS_KEY_ID=AKIXXXXXXXXXXA
+        export AWS_SECRET_ACCESS_KEY=qHZXXXXXXQc4oMQMnAOj+340XXxO2s
+        
+    To set these environment variables on Windows, use set :
+        
+        set AWS_ACCESS_KEY_ID=AKIXXXXXXXXXXA
+        set AWS_SECRET_ACCESS_KEY=qHZXXXXXXQc4oMQMnAOj+340XXxO2s
+    
+    > **Note**: If the application is running in an EC2 instance and credentials are not defined in the init configuration, the credentials will be obtained from the [IAM role](https://docs.amazonaws.cn/en_us/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html) assigned for the Amazon EC2 instance. This option is available only with Amazon SQS connector v2.0.0 and above.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md:88]: Please include this note as an [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=) to ensure consistency within the documentation.

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md [hunk 17]
```diff
@@ -0,0 +1,1058 @@
...
+        <tr>
+            <td>apiCallAttemptTimeout</td>
+            <td>The amount of time(in seconds) to wait for the http request to complete before giving up and timing out.</td>
+            <td>No</td>
+        </tr>
+    </table>
+
+    > **Note**: You can either pass credentials within the `init` configuration or set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` as environment variables. The AWS SDK uses provider chains to look for AWS credentials in system/user environment variables.
+        
+    To set these environment variables on Linux, macOS, or Unix, use export :
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md:78]: Shall we write the following as [content tabs](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/?h=tabs) for better clarity?

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md [hunk 18]
```diff
@@ -0,0 +1,1058 @@
...
+    <amazonsqs.createQueue configKey="AMAZON_SQS_CONNECTION">
+        <queueName>Queue</queueName>
+        <attributeEntries>"{'my_attribute_name_1': {'DataType': 'String','StringValue': 'my_attribute_value_1' }}"</attributeEntries>
+        <tags>"{QueueType: Testing}"</tags>
+    </amazonsqs.createQueue>
+    ```
+---
+
+??? note "deleteQueue"
+    This operation deletes the queue specified by the QueueUrl. See the [related API documentation](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_DeleteQueue.html) for more information.
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md:711]:
```suggestion
    This operation deletes the queue specified by `QueueUrl`. See the [related API documentation](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_DeleteQueue.html) for more information.
```

File: en/mkdocs.yml [hunk 19]
```diff
@@ -595,8 +595,12 @@ nav:
                   - AmazonS3 Reference: reference/connectors/amazons3-connector/1.x/amazons3-connector-1.x-reference.md
           - Amazon SQS Connector:
               - Amazon SQS Overview: reference/connectors/amazonsqs-connector/amazonsqs-connector-overview.md
-              - Amazon SQS Example: reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md
-              - Amazon SQS Reference: reference/connectors/amazonsqs-connector/amazonsqs-connector-config.md
+              - 2.x:
+                 - Amazon SQS Example: reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md
+                 - Amazon SQS Reference: reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md
+              - 1.x:
+                   - Amazon SQS Example: reference/connectors/amazonsqs-connector/1.x/amazonsqs-connector-1.x-example.md
```
Comment (chanikag) [inline en/mkdocs.yml:602]: Can you recheck the formatting here.

**Review by chanikag — APPROVED** [review]

File: en/mkdocs.yml [hunk 20]
```diff
@@ -595,8 +595,12 @@ nav:
                   - AmazonS3 Reference: reference/connectors/amazons3-connector/1.x/amazons3-connector-1.x-reference.md
           - Amazon SQS Connector:
               - Amazon SQS Overview: reference/connectors/amazonsqs-connector/amazonsqs-connector-overview.md
-              - Amazon SQS Example: reference/connectors/amazonsqs-connector/amazonsqs-connector-example.md
-              - Amazon SQS Reference: reference/connectors/amazonsqs-connector/amazonsqs-connector-config.md
+              - 2.x:
```
Comment (DinithiDiaz) [inline en/mkdocs.yml:598]: As discussed, please use the same format as for version 1.x and create a separate directory for version 2.x content. Also, rename the file to include '2.x' in the name.

Additionally, add redirects for the two URLs we will no longer be using to prevent 404 errors when switching between versions.

File: en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md [hunk 21]
```diff
@@ -0,0 +1,285 @@
...
+
+<img src="{{base_path}}/assets/img/integrate/connectors/amazonsqs-connector.jpg" title="AmazonSQS-Connector" width="800" alt="AmazonSQS-Connector"/>
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Set up the environment 
+
+1. Follow the steps mentioned in the [Setting up the Amazon S3 Environment]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document to create an Amazon account and obtain access key ID and secret access key. Keep them saved to use in the next steps.
+
+2. Download and place the following client libraries in to the <PRODUCT_HOME>/lib directory (From SQS connector 2.0.0 and above).
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md:19]:
```suggestion
2. Download and place the following client libraries in to the `<PRODUCT_HOME>/lib` directory (From SQS connector 2.0.0 and above).
```

File: en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-reference.md [hunk 22]
```diff
@@ -0,0 +1,1060 @@
...
+        
+        - To set these environment variables on Linux, macOS, or Unix, use export :
+        
+                export AWS_ACCESS_KEY_ID=AKIXXXXXXXXXXA
+                export AWS_SECRET_ACCESS_KEY=qHZXXXXXXQc4oMQMnAOj+340XXxO2s
+            
+        - To set these environment variables on Windows, use set :
+            
+                set AWS_ACCESS_KEY_ID=AKIXXXXXXXXXXA
+                set AWS_SECRET_ACCESS_KEY=qHZXXXXXXQc4oMQMnAOj+340XXxO2s
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-reference.md:87]:
```suggestion
!!! note
    You can either pass credentials within init configuration or set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` as environment variables. The AWS SDK uses provider chains to look for AWS credentials in system/user environment variables.

    To set these environment variables follow the below steps based on your operating system: 

    === "Linux, macOS, or Unix"
        ```    
        export AWS_ACCESS_KEY_ID=AKIXXXXXXXXXXA
        export AWS_SECRET_ACCESS_KEY=qHZXXXXXXQc4oMQMnAOj+340XXxO2s
        ```    
    === "Windows"
        ```    
        set AWS_ACCESS_KEY_ID=AKIXXXXXXXXXXA
        set AWS_SECRET_ACCESS_KEY=qHZXXXXXXQc4oMQMnAOj+340XXxO2s
        ```
```

File: en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md [hunk 23]
```diff
@@ -0,0 +1,285 @@
...
+
+<img src="{{base_path}}/assets/img/integrate/connectors/amazonsqs-connector.jpg" title="AmazonSQS-Connector" width="800" alt="AmazonSQS-Connector"/>
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Set up the environment 
+
+1. Follow the steps mentioned in the [Setting up the Amazon S3 Environment]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document to create an Amazon account and obtain access key ID and secret access key. Keep them saved to use in the next steps.
+
+2. Download and place the following client libraries in to the `<PRODUCT_HOME>/lib` directory (From SQS connector 2.0.0 and above).
```
Comment (SanojPunchihewa) [inline en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md:19]: Since this the 2.x doc page, do we need explicitly mention connector version here?

File: en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md [hunk 24]
```diff
@@ -0,0 +1,285 @@
...
+
+It has a single HTTP API resource, which is `sendToQueue`. 
+
+<img src="{{base_path}}/assets/img/integrate/connectors/amazonsqs-connector.jpg" title="AmazonSQS-Connector" width="800" alt="AmazonSQS-Connector"/>
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Set up the environment 
+
+1. Follow the steps mentioned in the [Setting up the Amazon S3 Environment]({{base_path}}/reference/connectors/amazonsqs-connector/amazonsqs-connector-config) document to create an Amazon account and obtain access key ID and secret access key. Keep them saved to use in the next steps.
```
Comment (SanojPunchihewa) [inline en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md:17]: This link doesn't contain any details about Amazon account creation or obtaining access keys. Instead shall we point to this doc? https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-setting-up.html

File: en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md [hunk 25]
```diff
@@ -0,0 +1,285 @@
...
+
+2. Download and place the following client libraries in to the `<PRODUCT_HOME>/lib` directory (From SQS connector 2.0.0 and above).
+   
+     - apache-client-2.20.26.jar
```
Comment (SanojPunchihewa) [inline en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md:21]: Better if we can add the maven links as well, so that the user can easily download them.

File: en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md [hunk 26]
```diff
@@ -0,0 +1,285 @@
...
+       --header 'Content-Type: application/json' \
+       --data '{
+          "companyName":"WSO2",
+          "queueName":"Queue1"
+       }'
+    ```
+
+**Expected Response**: 
+
+Next, you will receive a response with `sys_id`. Make sure to save it.
```
Comment (SanojPunchihewa) [inline en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md:260]: Shall we update this description? It seems irrelevant here

File: en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-reference.md [hunk 27]
```diff
@@ -0,0 +1,1060 @@
...
+    ```
+    
+    **Sample request**
+
+    ```xml
+    <amazonsqs.purgeQueue configKey="AMAZON_SQS_CONNECTION">
+        <queueUrl>https://sqs.us-west-2.amazonaws.com/882234390511/Queue</queueUrl>
+    </amazonsqs.purgeQueue>
+    ```
+---
```
Comment (SanojPunchihewa) [inline en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-reference.md:1060]: Shall we add the error codes as well? Please refer http://localhost:8000/en/latest/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-reference/#error-codes-related-to-microsoft-azure-storage-connector as an example

File: en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-reference.md [hunk 28]
```diff
@@ -0,0 +1,1060 @@
+# Amazon SQS Connector Reference
+
+The following operations allow you to work with the Amazon SQS Connector. Click an operation name to see parameter details and samples on how to use it.
+
+---
+
+## Initialize the connector
+
+To use the Amazon SQS connector, add the element in your configuration before carrying out any Amazon SQS operations. This Amazon SQS configuration authenticates with Amazon SQS by specifying the AWS access key ID and secret access key ID, which are used for every operation.
```
Comment (SanojPunchihewa) [inline en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-reference.md:9]: Adding the init element was necessary in the older connector model. In this 2.x series we are using a connection and the config key. Shall we update this description accordingly?

File: en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-reference.md [hunk 29]
```diff
@@ -0,0 +1,1060 @@
...
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>region</td>
+            <td>The regional endpoint to make your requests (e.g., us-east-1).</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>secretAccessKey</td>
+            <td>TA secret access key associated with your AWS access key.</td>
```
Comment (SanojPunchihewa) [inline en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-reference.md:31]:
```suggestion
            <td>The secret access key associated with your AWS access key.</td>
```

File: en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-reference.md (hunk 27 above)
Comment (kalaiyarasiganeshalingam) [reply en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-reference.md:1063]: Added.

File: en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md (hunk 26 above)
Comment (kalaiyarasiganeshalingam) [reply en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md]: Removed this.

File: en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md (hunk 23 above)
Comment (kalaiyarasiganeshalingam) [reply en/docs/reference/connectors/amazonsqs-connector/2.x/amazonsqs-connector-2.x-example.md]: Removed now.

**Review by DinithiDiaz — APPROVED** [review]

**Review by SanojPunchihewa — APPROVED** [review]
